### ISSUE TITLE
About Loss ?

### ISSUE THREAD

**Issue by huynth1801** [csqiangwen/DeepFillv2_Pytorch#12]
Hello thanks for sharing your work.
I am trying training the model with CelebA datasets but the loss function when training is too large. Do you think this loss function is problematic for me?
![image](https://user-images.githubusercontent.com/81891365/155912568-5a4fad46-0e84-4aa5-8643-744fce00454b.png)
when i tried the test, the result made me not satisfied
![result_0](https://user-images.githubusercontent.com/81891365/155912667-3831ef2e-ea4f-4c5e-bc40-693af90ef2b9.jpg)
can you help me ? Thank you
